roles/pulp_common/handlers/main.yml

```diff
@@ -4,19 +4,26 @@
 - name: Load the SELinux policy packages
   command: 'semodule -i /usr/local/share/selinux/{{ ansible_facts.selinux.type }}/{{ item }}.pp'
   loop: '{{ __pulp_selinux_policy_pkgs }}'
+  become: true
```

Comment:
Hmm, how the heck did these work previously? I think this may fix a recently reported issue by a user.

These are worthy of a changelog entry.

Reply:
The ansible.cfg in pulplift contains a "become=yes" that seemed wrong to me.

Reply:
Do we have a ticket?

Reply:
We have now:
https://pulp.plan.io/issues/7736